content/ytt/docs/latest/known-limitations.md

```diff
@@ -11,3 +11,4 @@ title: Known Limitations
     second: *content
     ```
     `second` key-value pair will _not_ contain 123 since YAML anchors are resolved before ytt evaluates templating directives.
+- Starlark mutliline strings do not preserve whitespace. Reason for this is because ytt uses a modified version of Starlark that is not whitespace sensitive in order to circumvent detecting errors in indentation during parsing. More information on why [here](lang.md).
```

Comment:
```suggestion
- Starlark multiline strings do not preserve whitespace. Reason for this is because ytt uses a modified version of Starlark that is not whitespace sensitive in order to circumvent detecting errors in indentation during parsing. More information on why [here](lang.md).
```

There's something formal about documentation: it's ideally authoritative. As a part of that formality there's a heightened expectation for precision. It's from _that_ place that this somewhat pedantic comment comes...

- _technically_, Starlark multiline strings _do_ preserve whitespace. So, to be more precise, let's shift the ideas a bit and make sure we keep connected the effect "multiline strings lose indentation" with the cause "`ytt` modified Starlark".
- this spot (i.e. "Known Limitations") is here to call out what's limited — it would be more "DRY" if we pull the rationale to the _other_ spot in the docs where we explain the situation more fully.
- speaking of which, I think you mean to point to the section you added in the other file in this commit; it's currently pointing at a very generic `lang.md`.

Reply:
Agreed, I updated this file to only contain mention of the limitation, then moved the details to `lang-ref-string.md` and linked to it.